src/adhocracy_frontend/adhocracy_frontend/static/js/i18n.ts

```diff
@@ -54,6 +54,7 @@ translateMe = [
     "TR__PROPOSALS",
     "TR__PUBLISH",
     "TR__RATES",
+    "TR__RESOURCE_COLLABORATIVE_TEXT_EDITING",
```

Comment:
This file is not actually used, but `extract_messages` will extract the strings nonetheless. This is useful if a string is not used in core, but you still want it to be translated there, e.g. because it is used in serveral projects.